src/components/base-address/BaseAddress.vue

```diff
@@ -0,0 +1,539 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+//
+
+<template>
+  <div class="base-address">
+    <!-- Display fields -->
+    <v-expand-transition>
+      <div v-if="!editing" class="address-block">
```

Comment:
I think that if you run `npm run lint`, it will auto-fix this. But watch for undesired changes.

Reply:
The GHA warning is now gone, but please run `npm run lint` before committing. It will reformat the HTML but this will ensure consistent style between components/repos.

Reply:
`npm run lint` did not make any changes to the HTML; however, I was able to go through and manually make the styling changes suggested by eslint.

Reply:
OK, thanks.

Maybe you're not using one of the eslint presets that auto-fixes the HTML...?

I'm not going to worry about this. Resolving.